Backend/src/v1/check.py

```diff
@@ -0,0 +1,22 @@
+import os

```

Comment:
The file `check.py` should not be in the folder `V1` as it contains no routes.

I do not see where it is used either. Could you also add a comment on how it is used? As it is a bit harder to backtrack files without functions.

Reply:
ah this was a test script that i added to check paths, will be removed.